StandaloneMmPkg/Drivers/MmCommunicationDxe/MmCommunicationDxe.c

```diff
@@ -25,6 +25,203 @@ EFI_MM_COMMUNICATION_PROTOCOL  mMmCommunication = {
 
 MM_COMM_BUFFER             mMmCommonBuffer;
 EFI_SMM_CONTROL2_PROTOCOL  *mSmmControl2;
+EFI_SMM_ACCESS2_PROTOCOL   *mSmmAccess;
+BOOLEAN                    mSmmLocked = FALSE;
+BOOLEAN                    mEndOfDxe  = FALSE;
+
+//
+// Data structure used to declare a table of protocol notifications and event
+// notifications required by the Standalone Mm environment
+//
+typedef struct {
+  BOOLEAN             Protocol;
+  BOOLEAN             CloseOnLock;
+  EFI_GUID            *Guid;
+  EFI_EVENT_NOTIFY    NotifyFunction;
+  VOID                *NotifyContext;
+  EFI_TPL             NotifyTpl;

```

Comment:
Remove this field as all are using TPL_CALLBACK

Reply:
Removed in latest code change